pFront/src/app/login/login.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { IpService } from '../ip.service';
+import { Ilogin } from '../../model/Ilogin';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrl: './login.component.scss'
+})
+export class LoginComponent {
+
+  constructor(private router: Router,private ip:IpService) { }
+
+  ngOnInit(): void {
+  }
+
+  logindata: Ilogin = {
+    username: '',
+    password: ''
+  
+  }
+
+ async loginUser(){
+
+    await fetch('http://'+this.ip.Ip+':8080/api/auth/login', {
```

Comment:
Proponuję zastosowanie stałych dla ścieżek - umożliwia to łatwiejsze modyfikowanie i dostęp